types/result.go

```diff
@@ -264,3 +265,21 @@ func ParseABCILogs(logs string) (res ABCIMessageLogs, err error) {
 	err = json.Unmarshal([]byte(logs), &res)
 	return res, err
 }
+
+var _, _ types.UnpackInterfacesMessage = SearchTxsResult{}, TxResponse{}
+
+// UnpackInterfaces implements UnpackInterfacesMessage.UnpackInterfaces
```

Comment:
I don't really think this adds much. What is actually happening during `types.UnpackInterfaces(tx, unpacker)`?

Reply:
Left a comment inline in the code. I'm not sure what's a better comment - all of the other interface methods have something similar (i.e. `implements sdk.Msg`)

Reply:
can we move the inline comment here instead?

Reply:
yep